README.md

```diff
@@ -470,12 +506,27 @@ The output is a list of statistics for each question from the reference Q&A data
 - `answer_relevance_error`: (optional) error message if answer relevance evaluation failed
 - `answer_relevance_cost`: The LLM use cost of computing `answer_relevance`, in US dollars
 - `actual_steps`: (optional) copy of the steps in the evaluation target, if specified there
-- `steps_score`: a real number between 0 and 1, computed by comparing the results of the last steps that were executed to the reference's last group of steps. If there is no match in the actual steps, then the score is `0`. Otherwise, it is calculated as the number of the matched steps on the last group divided by the total number of steps in the last group.
+- `steps_score`: a real number between 0 and 1, computed by comparing the results of the last executed steps to the output of the reference's last group of steps.
+    - If there is no match in the actual steps, then the score is `0.0`
+    - For executed steps whose name is "retrieval":
+        - If the last reference group contains a retrieval step and its output (the reference context) is specified, the score is the [recall at k](#context-recallk) of the retrieved document ids with respect to the reference.
```

Comment:
How can I specify the reference output? Should we provide an example?

Reply:
That is for [#19](https://github.com/Ontotext-AD/graphrag-eval/pull/19)